src/python/pants/backend/python/subsystems/python_setup.py

```diff
@@ -20,18 +20,14 @@ class PythonSetup(Subsystem):
   @classmethod
   def register_options(cls, register):
     super(PythonSetup, cls).register_options(register)
-    # TODO: On removal, make ['CPython>=2.7,<3'] the default for --interpreter-constraints.
-    register('--interpreter-requirement', advanced=True, default='CPython>=2.7,<3',
-             removal_version='1.5.0.dev0', removal_hint='Use --interpreter-constraints instead.',
-             help='The interpreter requirement string for this python environment.')
-    # Note: This will replace two options:
+    # Note: This replaces two options:
     # A) The global --interpreter option in the old python tasks.
     #    That flag is only relevant in the python backend, and should never have been
     #    global to begin with.
     # B) The --interpreter-requirement option above.  That flag merely served to set the
```

Comment:
This comment makes no sense now, since the options it mentions are now gone. Why did you choose to keep it?

Reply:
Just due to a lack of context as to whether leaving some of the history here might be useful to someone debugging something here.